docs/08-registration-rules.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Registration rules
+type: Details
+---
+
+This document presents the rules you must follow when you configure the Helm Broker. It also describes the Helm Broker's behavior in case of conflicts.
```

Comment:
```suggestion
This document presents the rules you must follow when you configure Helm Broker. It also describes Helm Broker's behavior in case of conflicts.
```